ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-ref.yaml

```diff
@@ -9,6 +9,12 @@ ref:
   - name: TEST_SUITE
     default: openshift/conformance/parallel
     documentation: The test suite to run.  Use 'openshift-test TEST_COMMAND --help' to list available suites.
+  - name: TEST_OPTIONS
+    default: ""
+    documentation: The test command options. Use 'openshift-test TEST_COMMAND --help' to list available options.
+  - name: OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE
```

Comment:
This should be a dependency, see the definition of the install one

Reply:
https://github.com/openshift/release/commit/0493745b0afad29fc76102c5bf87f633e2a4daa6